cognite_toolkit/_cdf_tk/load/_resource_loaders.py

```diff
@@ -2155,3 +2260,37 @@ def delete(self, ids: SequenceNotStr[WorkflowVersionId]) -> int:
             else:
                 successes += 1
         return successes
+
+    @classmethod
+    @lru_cache(maxsize=1)
+    def get_write_cls_parameter_spec(cls) -> ParameterSpecSet:
+        spec = super().get_write_cls_parameter_spec()
+        # The Parameter class in the SDK class WorkflowVersion implementation is deviating from the API.
+        # So we need to modify the spec to match the API.
+        parameter_path = ("workflowDefinition", "tasks", ANY_INT, "parameters")
+        length = len(parameter_path)
+        for item in spec:
+            if len(item.path) >= length + 1 and item.path[:length] == parameter_path[:length]:
+                # Add extra ANY_STR layer
+                # The spec class is immutable, so we use this trick to modify it.
+                object.__setattr__(item, "path", item.path[:length] + (ANY_STR,) + item.path[length:])
+        spec.add(ParameterSpec((*parameter_path, ANY_STR), frozenset({"dict"}), is_required=True, _is_nullable=False))
+        # The depends on is implemented as a list of string in the SDK, but in the API spec it
+        # is a list of objects with one 'externalId' field.
+        spec.add(
+            ParameterSpec(
+                ("workflowDefinition", "tasks", ANY_INT, "dependsOn", ANY_INT, "externalId"),
```

Comment:
I was a bit confused about the use of `ANY_INT` here because I expected it to validate if a parameter was a number or not. Here it seems to validate wether if it is an array of (ANY_INT) length? Is this some Pythonic way of using int, or should it be some (ANY_LIST) or something?